service/surf/settings/base.py

```diff
@@ -105,7 +104,6 @@
     'django.middleware.locale.LocaleMiddleware',
     'social_django.middleware.SocialAuthExceptionMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
We need to keep this. Not for the API, but for the admin

Reply:
✅